### ISSUE TITLE
Cannot build for Android after macOS platform merge

### ISSUE THREAD

**Issue by LinusU** [cloudwebrtc/flutter-webrtc#142]
**Describe the bug**

After #138 was merged I can no longer build for Android.

**To Reproduce**

1. Checkout a118213 (`git checkout a118213`)
1. Clean & run (`git clean -fdx`, `flutter run -d A10`)
1. See that everything works
1. Checkout 8dccca7 (`git checkout 8dccca7`)
1. Clean & run (`git clean -fdx`, `flutter run -d A10`)
1. See error below

```text
FAILURE: Build failed with an exception.

* Where:
Script '/Users/linus/coding/flutter/packages/flutter_tools/gradle/flutter.gradle' line: 653

* What went wrong:
A problem occurred configuring project ':path_provider_fde'.
> Could not get unknown property 'android' for project ':path_provider_fde' of type org.gradle.api.Project.
```

**Expected behavior**

For the build process not to error

**Platform information**

* **Flutter version**: Channel stable, v1.9.1+hotfix.6, on Mac OS X 10.15.1
* **Plugin version**: 8dccca7
* **OS**: Android
* **OS version**: 7.0 (API 24)

**Comment by LinusU** [comment]
I can get the build to work if I remove the dependency on `path_provider_fde`, but then whenever I try to do something in the example app, I get the following error (for different methods):

```text
E/flutter (28257): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: MissingPluginException(No implementation found for method createVideoRenderer on channel FlutterWebRTC.Method)
```

```text
I/flutter (26198): MissingPluginException(No implementation found for method createPeerConnection on channel FlutterWebRTC.Method)
```

**Comment by cloudwebrtc** [comment]
Sure it's a bug, I will fix it immediately.

**Comment by LinusU** [comment]
@cloudwebrtc thanks for the quick fix!

I'm still stuck on my second problem though, trying to do anything in the UI spits out `MissingPluginException`, and I cannot actually do anything.

I've just tried this on a totally clean checkout of 98a5d29

-----

I've figured out the problem, it's this change:

```diff
 flutter:
   plugin:
-    androidPackage: com.cloudwebrtc.webrtc
-    pluginClass: FlutterWebRTCPlugin
+    platforms:
+      android:
+        package: com.cloudwebrtc.webrtc
+        pluginClass: FlutterWebRTCPlugin
+      ios:
+        pluginClass: FlutterWebRTCPlugin
+      macos:
+        pluginClass: FlutterWebRTCPlugin
```

Reverting that patch makes my Android build again. I'm guessing that Flutter 1.9 doesn't support that new style...

I tried just adding back to removed lines, so that it has both the new style and the old style, and that seems to work 🙌 

Maybe we could have that until the next stable release of Flutter supports the new style?

**Comment by LinusU** [comment]
Fixed by 98a5d29 + #143 🎉
